jdaviz/configs/default/plugins/markers/markers.py

```diff
@@ -52,9 +58,9 @@ def user_api(self):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         if self.config == 'cubeviz':
-            headers = ['spectral_axis', 'spectral_axis:unit',
-                       'slice', 'pixel_x', 'pixel_y',
-                       'world_ra', 'world_dec', 'value', 'value:unit', 'viewer']
+            headers = ['spectral_axis', 'spectral_axis:unit', 'slice',
+                       'pixel_x', 'pixel_y', 'world_ra', 'world_dec',
+                       'value', 'value:unit', 'viewer']

```

Comment:
seems nothing has changed here besides spacing - for anything that isn't necessary to keep, I'd vote to remove these from the diff to make tracing back the history of changes easier.